src-tests/Tests.hs

```diff
@@ -19,14 +20,15 @@ arbitraryVersion = arbitraryBoundedEnum
 arbitraryHashOptions :: Gen HashOptions
 arbitraryHashOptions =
   do p <- arbitraryWithin 1 4
-     HashOptions <$> arbitraryWithin FFI.ARGON2_MIN_TIME (min FFI.ARGON2_MAX_TIME 65536)
+     HashOptions <$> arbitraryWithin FFI.ARGON2_MIN_TIME (min FFI.ARGON2_MAX_TIME (2 ^ 13))
```

Comment:
Even with this change, on my over 5 years old computer it takes 4 minutes. Keeping `2 ^ 16` would take around half an hour